data/campaigns/Eastern_Invasion/scenarios/17b_All-In.cfg

```diff
@@ -0,0 +1,1663 @@
+        {CHANGE_RECRUITS_IF_RICH 3 (Dark Adept,Walking Corpse,Soulless,Ghoul)        (Necromancer) }
+        {CHANGE_RECRUITS_IF_RICH 4 (Dark Adept,Vampire Bat,Blood Bat,Skeleton Rider) (Death Knight)}
+        {CHANGE_RECRUITS_IF_RICH 5 (Dark Adept,Vampire Bat,Blood Bat,Ghost)          (Nightgaunt)  }
+        {CHANGE_RECRUITS_IF_RICH 6 (Dark Adept,Walking Corpse,Soulless,Ghoul)        (Necromancer) }
+        {CHANGE_RECRUITS_IF_RICH 7 (Dark Adept,Skeleton,Skeleton Archer)             (Banebow)     }
+        {CHANGE_RECRUITS_IF_RICH 8 (Dark Adept,Walking Corpse,Soulless,Ghoul)        (Necromancer) }
+
+        {VARY_AI_BY_SCHEDULE (2,3,4,5,6,7,8)}
+    [/event]
+
+    [side]
+        side=9
+        color=brightgreen
+        type=Dark Adept
+        id=Lethin
+        name= _ "Lethin"
+        canrecruit=yes
+        facing=sw
+        [modifications]
+            {TRAIT_INTELLIGENT} # same traits as S03's Mal-Tar
+            {TRAIT_RESILIENT}
+        [/modifications]
+        recruit=Walking Corpse,Soulless,Ghoul #Skeletal Corpse,Skeletal Soulless
+        controller=ai
+        team_name=undead
+        user_team_name=_"Undead"
+        {GOLD 100 300 500}
+        income=-2
+        village_gold=0
+        {FLAG_VARIANT undead}
+
+        {LOYAL_UNIT 9 (Soulless) 50 11} {GUARDIAN}
+
+        [unit]
+            x,y,name=16,8,Rerivyan
+            type,canrecruit=Dark Adept,yes
+            gender,facing=female,se
+        [/unit]
+        {LOYAL_UNIT 9 (Walking Corpse) 20 10} {GUARDIAN}
+        {LOYAL_UNIT 9 (Ghoul) 15 10} {GUARDIAN}
+
+        [unit]
+            x,y,name=9,26,Vowyn
+            type,canrecruit=Dark Adept,yes
+            gender,facing=female,se
+        [/unit]
+        {LOYAL_UNIT 9 (Soulless) 9 22} {GUARDIAN}
+    [/side]
+    {RECRUIT_UNIT_VARIATIONS 9 "Walking Corpse" none,none,none,none}
+    {RECRUIT_UNIT_VARIATIONS 9 "Soulless"       none,none,none,none}
+    {SILENTLY_LIMIT_LEADER_MOVES 9 1}
+    [event]
+        name=prestart
+
+        [filter_condition]
+            {VARIABLE_CONDITIONAL mal_tar_alive equals yes}
+        [/filter_condition]
+
+        {CLEAR_VARIABLE mal_tar_alive}
+        {MODIFY_UNIT id=Lethin name _"Mal-Tar"}
+        {MODIFY_UNIT id=Lethin experience 29}
+        [event]
+            name=last breath
+            [filter]
+                id=Lethin
+            [/filter]
+            # really Mal-Tar
+            [message]
+                speaker=unit
+                message= _ "Wait, wait, stop, I surrender! I know your leader Gweddry!"
+            [/message]
+            [message]
+                speaker=unit
+                message= _ "He defeated me back in the Estmarks but still let me live! Let me go and I promise you’ll never see me again!"
+            [/message]
+            [message]
+                speaker=second_unit
+                message= _ "I don’t know... let me ask the king."
```

Comment:
Capitalise 'king'?

Reply:
I disagree. It's not a name or anything, just a noun.